src/pages/CartPage.js

```diff
@@ -7,7 +8,12 @@ class cartPage extends Component {
     const products = JSON.parse(localStorage.getItem('products') || '[]');
     return (
       <div data-testid="shopping-cart-empty-message">
-        {products.length > 0 ? <CardCart products={ products } /> : <EmptyCart />}
+        <Link to="/checkout" data-testid="checkout-products">Checkout</Link>
+        {
+          products.length > 0
+            ? <CardCart products={ products } />
```

Comment:
```suggestion
        {
          products.length > 0
            ?
              <div>
                 <Link to="/checkout" data-testid="checkout-products">Checkout</Link>
                 <CardCart products={ products } />
              </div>
```
Não seria interessante colocar o link para o checkout somente se tivesse algum item no carrinho? :thinking:

Reply:
Brabo